ProductionModePlugin.js

```diff
@@ -67,34 +67,37 @@ class ProductionModePlugin {
         if(param.isString() && param.string === "globalize" && this.moduleFilter(request) &&
           !(globalizeCompilerHelper.isCompiledDataModule(request))) {
 
-          // Statically extract Globalize formatters and parsers from the request
-          // file only. Then, create a custom precompiled formatters/parsers module
-          // that will be called instead of Globalize, which in turn requires
-          // Globalize, set the default locale and then exports the Globalize
-          // object.
-          const compiledDataFilepath = globalizeCompilerHelper.createCompiledDataModule(request);
+          this.supportedLocales.forEach((locale) => {
```

Comment:
Please, also can you elaborate more about the motivations for this change? It's not clear to me why you need to iterate over all the locales here. At this point, the idea is that formatters and parsers are extracted regardless of the locale and below (i.e., `fnContent = globalizeCompilerHelper.compile(locale)`), all of them are compiled for each supported locale.